package.json

```diff
@@ -8,8 +8,15 @@
     "install-be": "cd ./game-backend && npm install",
     "build-fe": "cd ./game-frontend && cross-env NODE_ENV=production npm run build",
     "build-be": "cd ./game-backend && cross-env NODE_ENV=production npm run build",
+    "start-fe": "cd ./game-frontend && cross-env NODE_ENV=production npm run start",
+    "start-be": "cd ./game-backend && cross-env NODE_ENV=production npm run start",
     "test-fe": "cd ./game-frontend && cross-env NODE_ENV=test CI=true npm run test",
     "test-be": "cd ./game-backend && cross-env NODE_ENV=test CI=true npm run test",
+    "start-app": "npm run start-be && npm run start-fe",
```

Comment:
Is this script working? The `start-be` won't exit and therefore `start-fe` won't be executed.

Reply:
It is working, but don't ask me why 😅

Reply:
The `start-be` is working, but `start-fe` is not being executed on my machine.
Is it possible that you're running the frontend in another terminal session already?

Reply:
That was the case ^^
Fixed now